app/javascript/components/enrollment/steps/Exposure.js

```diff
@@ -251,35 +243,16 @@ class Exposure extends React.Component {
   };
 
   updateSOandNRSValidations = patient => {
-    if (!patient.symptom_onset && !patient.no_reported_symptoms) {
+    if (patient.symptom_onset && patient.asymptomatic) {
       schema = yup.object().shape({
         ...staticValidations,
         symptom_onset: yup
           .date('Date must correspond to the "mm/dd/yyyy" format.')
-          .max(
-            moment()
-              .add(30, 'days')
-              .toDate(),
-            'Date can not be more than 30 days in the future.'
-          )
-          .required('Please enter a Symptom Onset Date') // NOTE: replace this line with the commented out line below when first positive lab is enabled
-          // .required('Please enter a Symptom Onset Date OR select No Reported Symptoms and enter a first positive lab result')
-          .nullable(),
-        no_reported_symptoms: yup.bool().nullable(),
-      });
-    } else if (!patient.symptom_onset && patient.no_reported_symptoms) {
-      schema = yup.object().shape({
-        ...staticValidations,
-        symptom_onset: yup
-          .date('Date must correspond to the "mm/dd/yyyy" format.')
-          .oneOf([null, undefined])
-          .nullable(),
-        no_reported_symptoms: yup
-          .bool()
-          .oneOf([true])
+          .oneOf([null, undefined], 'Please enter a Symptom Onset Date OR select Asymptomatic, but not both.')
```

Comment:
This validation doesn't seem to be working as intended. I can enroll without specifying a SOD or Asymptomatic/Lab Result.

Reply:
maybe I misunderstood the requirements from Kelly, but I was under the impression there might be cases enrolled where symptom onset date is unknown, but the case is not necessarily asymptomatic. I'll reach out to Kelly and clarify tomorrow morning

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/2223a5e68c74a7d041ca0fb9b499b29e87662326

Reply:
thanks @tstrass , forgot to comment with the commit